code/datums/wires/airlock.dm

```diff
@@ -192,6 +213,15 @@
 		if(WIRE_ZAP1, WIRE_ZAP2) // Ouch.
 			if(isliving(usr))
 				A.shock(usr, 50)
+		if(WIRE_UNRESTRICTED_EXIT) // If you cut this wire, the unrestricted helper goes way. If you mend it, it'll go "haywire" and pick a new direction at random. Might have to cut/mend a time or two to get the direction you want.
+			if(!A.unres_sensor) //only works if the "sensor" is installed (a variable that we assign to the door either upon creation of a door with unrestricted directions or if an unrestricted helper is added to a door in mapping)
+				return
+			if(!mend)
+				A.unres_sides = NONE
+				A.update_appearance()
+			else
+				A.unres_sides = pick(NORTH, SOUTH, EAST, WEST)
+				A.update_appearance()
```

Comment:
```suggestion
			if(mend)
				A.unres_sides = pick(NORTH, SOUTH, EAST, WEST)
				A.update_appearance()
			else
				A.unres_sides = NONE
				A.update_appearance()

```
Just so they're more readable since there's no early return here.

Also do we _want_ them to be randomized between the four sides, seems kinda odd instead of just (an example) restoring their initial one.

Reply:
I wanted there to be some sort of way you could still use existing doors and leverage their "sensors" in the case of like maint bases, like if you really wanted to shuffle around some walls on a door already with a sensor (I.E.: change it from North to East), you won't have to tear down and create a whole new door.